definitions/marts/bigquery_marts/npq/npq_enrolments.sqlx

```diff
@@ -46,6 +46,8 @@ config {
         latest_lead_provider_name: "Name of Lead Provider associated with participant's latest declaration.",
         latest_declaration_date: "Date evidence was submitted for participant's latest declaration.",
         latest_declaration_updated_at: "Date participant's latest declaration was updated",
+        latest_declaration_delivery_partner_id: "The ID of the delivery partner that submitted the latest declaration, for declarations from the 2024 cohort onwards (excluding international applications); some providers may also include this for new declarations from earlier cohorts.",

```

Comment:
Don't change this for now as it's scope creep but I think structuring these as STRUCTs would be tidier. i.e. latest_declaration.delivery_partner.name. @consult1815 what do you think? Worth a separate Trello card?